core/src/main/java/org/apache/druid/data/input/impl/DelimitedParseSpec.java

```diff
@@ -123,6 +124,12 @@ public int getSkipHeaderRows()
     );
   }
 
+  @Override
+  public InputFormat toInputFormat()
```

Comment:
Where is the test that makes sure TSV is handled properly now? Also do we already have a similar test for CSV?

Reply:
What do you mean by "handling TSV properly"? This method does nothing useful for now. I think we should remove this method.

Reply:
Sorry, my comment was more about adding integration tests for streaming ingestion with the various formats, which I believe we're missing. This can be addressed in a follow up PR that tackles the issue comprehensively.

Reply:
Sounds good. I will add some tests with TSV in a follow-up PR.